source/beamline/BioXAS/BioXASBeamline.cpp

```diff
@@ -338,6 +338,19 @@ CLSValves* BioXASBeamline::valves() const
 	return result;
 }
 
+AMXRFDetector *BioXASBeamline::xrfDetector() const
```

Comment:
VESPERS already has a configuration where multiple XRF detectors can be used at once.